roles/lvm/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2025 VEXXHOST, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Discover lvm devices
+  ansible.builtin.shell: |
+    set -o pipefail
+    /sbin/pvdisplay | awk '/PV Name/ {print $3}' | sed 's/\/dev\///g'
+  register: lvm_devices
+  changed_when: lvm_devices.rc != 0
+  failed_when: false
```

Comment:
The issue here is that if we have an environment where this issue has already manifested, Atmosphere won't be able to fix it and instead add those PVs into it, which we don't want.